### DISCUSSION TITLE
What is exactly YourTts

### DISCUSSION THREAD

**Discussion by Ca-ressemble-a-du-fake** [coqui-ai/TTS#2014] [General Q&A] [answered]
Hi,

Although I have been following this project and using it  for months I am still a bit confused about what YourTts actually is. Part of my confusion is due to the fact that to test YourTTS one has to install CoquiTTS (see this [introduction to YourTTS](https://youtu.be/sCXOUhnfzxM?t=518)). So I am using one for the other and this may not be right.

I understood that Coqui TTS is a company that made VITS model and released it as open source. => This post explains it in a much better way : https://www.reddit.com/r/VocalSynthesis/comments/vw7ika/check_out_coqui_tts/

However I did not get what their business model is. EDIT => Coqui is funded by an ["investment company"](https://gitter.im/coqui-ai/TTS?at=632dc626f4d7a323dee89c83).

Regarding YourTts ~~is it right to consider it as a framework for different neural TTS models ? What is the goal of such framework, are there any other such frameworks or is YourTts the single one ?~~ it is an improvement of VITS (see @lexkoro answer below) and it uses Coqui framework.

Moreover is YourTts used internaly at Coqui's or a more advanced version is used ? 
Thank you for your explanations😊

**Comment by lexkoro** [comment, marked as answer]
In short: YourTTS is a modified version of the original VITS model provided here https://github.com/jaywalnut310/vits

YourTTS additionally adds multilingual support and training using dvectors. You can check the multilingual vits recipe which is basically the YourTTS model. https://github.com/coqui-ai/TTS/blob/dev/recipes/multilingual/vits_tts/train_vits_tts.py

**Reply by Ca-ressemble-a-du-fake** [reply]
Thank you @lexkoro that's clearer ! Now I get it for YourTTS and VITS ... but why do they provide Tacotron+/-DDC, GLOW, ... models ? For example @thorstenMueller trained DDC model with YourTTS. That's the crossroad (VITS-OTHER-MODELS) where I get confused.

**Reply by thorstenMueller** [reply]
Hmm, now i'm a little bit confused too, @Ca-ressemble-a-du-fake  ;-).
Where did you see that i trained a YourTTS model? I just trained a "normal" Tacotron2 DDC model with Coqui.

**Reply by Ca-ressemble-a-du-fake** [reply]
Thanks for the clarification @thorstenMueller ! I misunderstood everything, because so far for me the tts part of coqui == yourtts! So coqui is the framework for training (and infering) with whichever model (DDC in your case, bur you also did a ViTS one, right)?

**Reply by WeberJulian** [reply]
Actually the recipe is just the multilingual part, no recipe for YourTTS has been released. But you can make one from the released YourTTS config :)

**Reply by WeberJulian** [reply]
Coqui TTS is a GitHub repository where we implement various TTS architectures.

**Reply by WeberJulian** [reply]
For more information about the company go to https://coqui.ai/

**Reply by Ca-ressemble-a-du-fake** [reply]
Oh great thank you @WeberJulian, what a beautiful idea! You mean doing a training from the [v0.6.1 released config.json](https://coqui.gateway.scarf.sh/v0.6.1_models/tts_models--multilingual--multi-dataset--your_tts.zip), right ? It smells like new discussions will arise soon on the topic "Training YourTTS /speaker  embeddings / d vectors ..." :wink:
